app/Models/QueryBuilders/SearchFormQueryBuilder.php

```diff
@@ -111,16 +145,35 @@ public function buildQuery(): array
         $orQueries = [];
         $andQueries = [];
         $boolQueries = [];
+        $dateRange = [];
+
+        if (isset($this->startDate) && !empty($this->startDate)) {
```

Comment:
Do we need both isset and empty here?

Reply:
Yes, as I already mentioned above,
"isset" checks if user want to search for date
"empty" checks if date is not empty string ''.

Reply:
Does the user send empty string for search from front end?
I think once it is set, it will not be empty string.
Also if it is not empty, we can see it is set.

Reply:
Removed check for "isset".